udata_front/theme/gouvfr/datagouv-components/src/components/InformationPanel/InformationPanel.vue

```diff
@@ -0,0 +1,211 @@
+                >
+                    <template v-if="harvestExpanded">
+                        {{ $t('Close details') }}
+                    </template>
+                    <template v-else>
+                        {{ $t('See harvest') }}
+                    </template>
+                </button>
+            </div>
+        </header>
+        <div
+            class="accordion-content"
+            ref="harvestRef"
+        >
+            <div class="fr-pb-3w fr-mb-3w border-bottom border-default-grey">
+                <div class="fr-grid-row fr-grid-row--gutters fr-text--sm fr-m-0">
+                    <div v-for="(value, key) in props.dataset?.harvest" :key="key" class="fr-col-12 fr-col-sm-6 fr-col-md-4">
+                        <h3 class="subtitle fr-mb-1v">{{ key }}</h3>
+                        <p class="fr-text--sm fr-m-0 text-mention-grey ">{{ value }}</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+<script setup lang="ts">
+import { ref, onMounted } from 'vue';
+import type { Dataset, DatasetV2 } from "../../types/datasets";
+import CopyButton from "../CopyButton/CopyButton.vue";
+import { toggleAccordion } from "../../helpers/toggleAccordion";
+import { formatDate } from '../../helpers/index';
+import { getGranularity, fetchGranularities } from '../../api/granularity';
+import { getFrequencies, fetchFrequencies } from '../../api/frequency';
+import { fetchZone } from '../../api/zones';
+import useOEmbed from '../../composables/useOEmbed';
+import type { Frequencies } from '../../types/frequency';
+import type { Granularities } from '../../types/granularity';
+import type { License } from '../../types/licenses';
+const props = defineProps<{
+    dataset: DatasetV2 | Dataset,
+    license: License
+}>();
+const embedText = useOEmbed('dataset', props.dataset.id);
+const extrasExpanded = ref(false);
+const extrasRef = ref<HTMLDivElement | null>(null);
+const granularities = ref<Granularities>([]);
+const frequencies = ref<Frequencies>([]);
+const zone = ref<string | null>(null)
+const harvestExpanded = ref(false);
+const harvestRef = ref<HTMLDivElement | null>(null);
+const textAreaRef = ref<HTMLTextAreaElement | null>(null);
+function selectContent() {
+  if (textAreaRef.value) {
+    textAreaRef.value.select();
+  }
+};
+
+function expand() {
+  harvestExpanded.value = !harvestExpanded.value;
+  if(harvestRef.value) {
+    toggleAccordion(harvestRef.value, harvestExpanded.value);
+  }
+};
+
+function extrasExpand() {
+  extrasExpanded.value = !extrasExpanded.value;
+  if(extrasRef.value) {
+    toggleAccordion(extrasRef.value, extrasExpanded.value);
+  }
+};
+
+function hasExtras() {
+  return Object.keys(props.dataset.extras).length > 0;
+};
+
+function setZone() {
+  if (props.dataset.spatial?.zones?.length > 0) {
+    fetchZone(props.dataset.spatial.zones[0]).then(foundZone => zone.value = foundZone);
+  } else {
+    zone.value = null;
```

Comment:
This `else` case seems unnecessary. zone is already null on setup and we call this only during onMounted